packages/opentelemetry-basic-tracer/karma.conf.js

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const webpackConfig = require('./webpack/test.config.js');
+
+module.exports = (config) => {
```

Comment:
Since this file is duped of the one in core (https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-core/karma.conf.js), this and the core one can be refactored to extend off of a base config.